content/integrations/aws.md

```diff
@@ -103,7 +104,8 @@ There are two integration methods that can be used to allow Datadog to monitor y
                 "sns:Publish",
                 "sqs:GetQueueAttributes",
                 "sqs:ListQueues",
-                "sqs:ReceiveMessage"
+                "sqs:ReceiveMessage",
+                "support:*"
```

Comment:
we cannot go lower than that level (http://docs.aws.amazon.com/IAM/latest/UserGuide/list_trustedadvisor.html)